qpc/cli.py

```diff
@@ -63,18 +63,18 @@
 )
 
 
-class VersionGitAction(argparse.Action):
+class BuildShaAction(argparse.Action):
     """Action to show the current git commit SHA-1."""
 
     def __init__(
         self,
         option_strings,
         dest=argparse.SUPPRESS,
         default=argparse.SUPPRESS,
-        help="show program's current git commit SHA-1 and exit",
+        help="show git commit SHA-1 used to build the program and exit",
```

Comment:
I realize in hindsight that this is never shows since we hide the command from `--help`, but I'm leaving it here in case we ever change our mind.